src/features/projectsV2/ProjectDetails/styles/ProjectInfo.module.scss

```diff
@@ -30,6 +30,10 @@
   color: #2f3336; //to be replaced
   font-size: $fontXXSmall;
 }
+
+.keyInfoContainer {
+  gap: 16px;
+}
 .additionalInfoContainer {
   background: rgba(var(--more-info-background-color-new), 0.1);
   gap: 16px;
```

Comment:
Minor refactor: 
1. The gap is common for keyInfoContainer and additionalInfoContainer, so perhaps that style can be kept in a common place